app/javascript/Pages/Connections.jsx

```diff
@@ -1,40 +1,104 @@
 import React from "react";
+import axios from "axios";
 import PropTypes from "prop-types";
-import { Head, Link } from "@inertiajs/inertia-react";
+import { Inertia } from "@inertiajs/inertia";
 import Layout from "./Layout";
+import { useState } from "react";
+import { FaTimes } from "react-icons/fa";
 
 export default function Welcome(props) {
-  const ConnectionsList = (props) => {
-    const connectionsListChild = props.connections.map((c) => {
-      if (c.match.student) {
-        return <li>Student: {c.match.student.username}</li>;
-      } else {
-        return <li>Teacher: {c.match.skill_class.teacher.username}</li>;
-      }
-    });
+ const avatar = (image) => {
+  return `${image}?tr=w-60,h-60,r-max`;
+ };
+
+ const endConnectionHandler = (connection_id) => {
+  axios
+   .put(`/api/connections/${connection_id}/`, {
+    status: "given",
+    headers: window.defaultHeaders,
```

Comment:
You can also set the headers globally
https://github.com/axios/axios#global-axios-defaults